app/helpers/application_helper.rb

```diff
@@ -276,16 +276,16 @@ def modal_image(photo, options = {})
     )
   end
   
-  def stripped_first_paragraph_of_text(text)
+  def stripped_first_paragraph_of_text(text,split)
```

Comment:
Doesn't look like this broke anything, but it's generally preferable to make optional args like this optional, e.g. `stripped_first_paragraph_of_text( text, split = nil )`, then set a default value for `split` in the method.